README.md

```diff
@@ -1,5 +1,8 @@
 # ibm-vpc-file-csi-driver
 
+[![Build Status](https://api.travis-ci.com/IBM/ibm-vpc-file-csi-driver.svg?branch=master)](https://travis-ci.com/IBM/ibm-vpc-file-csi-driver)
```

Comment:
I think the new endpoint is v3.travis

Reply:
> I think the new endpoint is v3.travis

Ok we need to check but this is working for VPC file library